File: Belajar/Input_Output/PHP/struk_barang.php
<?php

$nama = readline("Masukkan nama barang: ");
$harga = (float)readline("Masukkan harga barang: ");
$jumlah = (int)readline("Masukkan jumlah barang: ");

$total = $harga * $jumlah;

$minimal_diskon = 40800;
$diskon_persen = 8;

if ($total > $minimal_diskon) {
    $diskon = $total * ($diskon_persen / 100);
} else {
    $diskon = 0;
}

$total_bayar = $total - $diskon;

echo "\n==== Struk Pembelian ====\n";
echo "Nama Barang   : " . $nama . "\n";
echo "Harga Satuan  : Rp " . intval($harga) . "\n";
echo "Jumlah        : " . $jumlah . "\n";
echo "Total         : Rp " . intval($total) . "\n";
echo "Minimal Diskon: Rp " . $minimal_diskon . "\n";
echo "Diskon (%)    : " . $diskon_persen . "%\n";
echo "Diskon        : Rp " . intval($diskon) . "\n";
echo "Total Bayar   : Rp " . intval($total_bayar) . "\n";
echo "=========================\n";

File: Belajar/Input_Output/C++/struk_barang.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    string nama;
    double harga;
    int jumlah;

    cout << "Masukkan nama barang: ";
    getline(cin, nama);

    cout << "Masukkan harga barang: ";
    cin >> harga;

    cout << "Masukkan jumlah barang: ";
    cin >> jumlah;

    double total = harga * jumlah;

    int minimalDiskon = 40800;
    int diskonPersen = 8;

    double diskon;
    if (total > minimalDiskon) {
        diskon = total * (diskonPersen / 100.0);
    } else {
        diskon = 0;
    }

    double totalBayar = total - diskon;

    cout << "\n==== Struk Pembelian ====\n";
    cout << "Nama Barang   : " << nama << endl;
    cout << "Harga Satuan  : Rp " << (int)harga << endl;
    cout << "Jumlah        : " << jumlah << endl;
    cout << "Total         : Rp " << (int)total << endl;
    cout << "Minimal Diskon: Rp " << minimalDiskon << endl;
    cout << "Diskon (%)    : " << diskonPersen << "%" << endl;
    cout << "Diskon        : Rp " << (int)diskon << endl;
    cout << "Total Bayar   : Rp " << (int)totalBayar << endl;
    cout << "=========================\n";

    return 0;
}
